### PR TITLE
@86eq301bj #Added CAA Reviewer Page UI

### PR DESCRIPTION
**Pull request by be3jaay**
image
image
image

### CODE REVIEW

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Box, Card } from '@mui/material';
+
```
Comment (JezChristian) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerBlock.tsx:2]: Remove space

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { Box, Card } from '@mui/material';
```
Comment (JezChristian) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerBlock.tsx:1]: We have an existing Card component in our core-library

**Review by JezChristian — CHANGES_REQUESTED** [review]

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerData.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+export const ContentReviewerData = [{
```
Comment (JMSevilla) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerData.ts:1]: Are these mock data referenced by our API ?

Comment (be3jaay) [reply packages/core-library/system/app/internal/blocks/Hub/content/approval/blocks/rqc/ContentReviewer/ContentReviewerData.ts:1]: For now, this data is not referenced by the API, I just based it on the Figma design under Regular Question. If ever I need to changed it, may I know the API endpoint,

**Review by JezChristian — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

**Pull request merged**